infra/modules/azure_app_service/app_service_plan.tf

```diff
@@ -0,0 +1,12 @@
+resource "azurerm_service_plan" "this" {
+  count = local.app_service_plan.enable ? 1 : 0
```

Comment:
may you add a comment on why we can disable service plan?

Reply:
I think it's described as variable documentation: https://github.com/pagopa/dx/pull/26/files#diff-8f8180c01ecb10d9bb9bed1f97e62b618a83b7b232d939e01a8cdd95182b458dR24